app-emulation/qemu/qemu-10.0.0.ebuild

```diff
@@ -54,7 +54,7 @@ DESCRIPTION="QEMU + Kernel-based Virtual Machine userland tools"
 HOMEPAGE="https://www.qemu.org https://www.linux-kvm.org"
 
 LICENSE="GPL-2 LGPL-2 BSD-2"
-SLOT="0"
+SLOT="0/$(ver_cut 1)"
```

Comment:
I don't see any value in this, you can already do `=qemu-9.1*` etc.

Reply:
1) Duplication of information
2) May make people wrongly think QEMU provides libraries and they should depend on it with :=